webpack.config.babel.js

```diff
@@ -58,6 +59,7 @@ module.exports = {
         new webpack.ProvidePlugin({
             [options.jsxPragma]: path.join(__dirname, "utils", "element"),
         }),
+        new webpack.optimize.UglifyJsPlugin({compress: {warnings: false}})
     ],
-    postcss: [veryimportant],
+    postcss: [veryimportant, autoprefixer],
```

Comment:
Looks like we'll need our browser config from above here. That is, `{browsers:['> 1%']}` removed from above on line 62 and passed into the `autoprefixer` plugin like so:

```
[veryimportant, autoprefixer({ browsers: ["> 1%"] })]
```

I think that will do the trick!